plugins/transfer/transfer.plugin.zsh

```diff
@@ -0,0 +1,22 @@
+transfer() {
+    if [ $# -eq 0 ]; then
+              echo "No arguments specified. Usage:"
+              echo "transfer /tmp/test.md"
+              echo "cat /tmp/test.md | transfer test.md"
+              return 1
+    fi
+ 
+    # write to output to tmpfile because of progress bar
+    tmpfile=$( mktemp -t transferXXX );
```

Comment:
Could `$tmpfile` and `$basefile` be made `local` to avoid polluting the caller's variable space?